styles/blocks/_gallery.scss

```diff
@@ -1202,32 +979,83 @@ $fotorama-thumb-arrow: 30px;
     }
 }
 
-.fotorama__thumb__arr {
+.fotorama__thumb--icon {
     @extend .fotorama-sprite;
-    background-size: 300%;
     font-size: 0.001px;
     padding-bottom: $fotorama-thumb-arrow;
 }
 
+.fotorama__thumb__arr--left {
+    .fotorama__thumb--icon {
+        background-position: -25px -265px;
+    }
+}
+
+.fotorama__thumb__arr--right {
+    .fotorama__thumb--icon {
+        background-position: -25px -350px;
+    }
+}
+
 .magnify-fullimage {
     display: none;
 }
 
+.fotorama__arr,
+.fotorama__thumb__arr {
+    @include fotorama-button-background();
+}
+
+.fotorama__wrap:not(.fotorama__wrap--toggle-arrows) {
+    .fotorama__fullscreen-icon,
+    .fotorama__zoom-out,
+    .fotorama__zoom-in {
+        @include fotorama-button-background();
+    }
+}
+
+// While first time init
 .gallery-placeholder {
     .loading-mask {
         padding: 0 0 50%;
-        position: absolute;
+        position: static;
     }
 
     .loader img {
         position: absolute;
     }
 }
 
-body.fotorama__fullscreen {
-    overflow-y: hidden;
+// Styles for spinner in gallery.
+.fotorama__spinner {
+    background-image: url('#{$baseDir}../images/loader-1.gif');
```

Comment:
I was getting a 404 for loader-1.gif. After updating this line it worked. I think this should be:
background-image: url('#{$baseDir}images/loader-1.gif');